src/main/java/com/example/petbuddybackend/entity/care/CareStatus.java

```diff
@@ -0,0 +1,12 @@
+package com.example.petbuddybackend.entity.care;
+
+public enum CareStatus {
+
+    PENDING,
+    ACCEPTED,
+    REJECTED,
+    CANCELLED,
+    OUTDATED,
+    PAID
+
+}
```

Comment:
- W sumie czym się cancelled od rejected różni? 
- Może by dodać AWAITING_PAYMENT żeby dać znać obu stronom że trzeba przelew zrobić?
- outdated to oznacza że usługa zakończona? Bo jeśli tak to dałbym COMPLETED

Reply:
Outdated to znaczy że nikt nie zatwierdził a data obecną już jest po dacie początku opieki(albo dzien przed, do ustalenia)

Reply:
Awaiting paymeny można dać
Rejected to że opiekun ksrzucil a cancelled to że klient jednak się wycofał, chyba bardziej pasuje ten rozdział do obu osobno, ale można ujednolicic